schema/ethernet.yml

```diff
@@ -18,6 +18,13 @@ properties:
       - LAN*
       - WAN*
       - "*"
+  name:
+    description:
+      This is a free text field, stating the administrative name of the
+      port. It may contain spaces and special characters, not exceeding 64 characters.
+    type: string
+    examples:
```

Comment:
I think examples should not be wrapped into quote marks?

Reply:
I am not sure about this, happened to refer few other places and did same way, while most other places seems to be without the quotes. So will go with your comment and remove the wraps.